File: HackerRank/30-days-of-code-challenges/js/day2.js
function processData(input) {
    //Enter your code here
    var data = input.split('\n');
    var mealCost = parseFloat(data[0]);
    var tipPercent = parseFloat(data[1]);
    var taxPercent = parseFloat(data[2]);
    var tip = mealCost * (tipPercent/100);
    var tax = mealCost * (taxPercent/100);
    var totalCost = Math.round(mealCost + tip + tax);

    console.log("The total meal cost is " + totalCost + " dollars.");
}

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

File: HackerRank/30-days-of-code-challenges/C/day2.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main() {

    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    double mealCost;
    double tipPercent;
    double taxPercent;
    scanf("%lf\n %lf\n %lf\n", &mealCost, &tipPercent, &taxPercent);

    double tip = mealCost * (tipPercent/100);
    double tax = mealCost * (taxPercent/100);
    int totalCost = round(mealCost + tip + tax);

    printf("The total meal cost is %d dollars.", totalCost);

    return 0;
}
